### PR TITLE
WIP: Fix get_code_params for Python 3.10

### PR DESCRIPTION
**Pull request by hodgestar**
In #43 and #37 it was reported that the arguments to `code(...)` have changed in Python 3.10. This affects the `build_code_chunk` and `get_code_params` functions in `genshi.compat`.  `build_code_chunk` was fixed in #49. This PR is intended to fix `get_code_params`.

WIP: Currently this PR only attempts to get the tests to fail.

### CODE REVIEW

**Comment by hodgestar** [comment]
image

So it looks like the transition to GitHub actions will have to happen.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [2.7, 3.6, 3.7, 3.8, 3.9, '3.10-dev', pypy2, pypy3]
+        python-version: [2.7,  3.4, 3.5, 3.6, 3.7, 3.8, 3.9, '3.10-dev', '3.11-dev', pypy2, pypy3]
```
Comment (jun66j5) [inline .github/workflows/tests.yml:10]: Python 3.4 on ubuntu-latest (20.04) is not available but available on ubuntu-18.04.

See:
* jun66j5/genshi@9f462c3
* https://github.com/jun66j5/genshi/actions/runs/1254667357
* https://github.com/actions/python-versions/blob/main/versions-manifest.json

Comment (FelixSchwarz) [reply .github/workflows/tests.yml:10]: Assuming we want to continue supporting Python 3.4/3.5 jun66j5/genshi@9f462c3 is a good approach. jun66j5/genshi@5908c14d should be replaced by `pip install -e .` or `python setup.py develop` though. Would you mind submitting a PR?

**Comment by hodgestar** [comment]
I'm happy to drop CI for Python 3.4 and 3.5 for now and get_code_params was fixed elsewhere, so closing this PR.

**Pull request closed without merging**